src/js/tabs/map.js

```diff
@@ -82,6 +100,39 @@ export function initMap() {
 
     map.addLayer(currentPositionLayer);
 
+    // Hide all layers initially except OSM
+    googleSatLayer.setVisible(false);
+    googleHybridLayer.setVisible(false);
+
+    $('#mapview').click(function (e) {
+        switch (e.target.text) {
```

Comment:
Can we instead rely on something more stable than this? These labels can end up being translated at some point. Could add things like `value` if we'd use buttons instead of anchor tags

Reply:
Code suggestions please :)

Reply:
Fixed.